README.md

```diff
@@ -61,6 +61,64 @@ Or with Nix 2.4+:
     $ nix profile install github:cachix/cachix/latest
 ```
 
+### Dev setup for VS Code
+
+1. [Enable flakes](https://nixos.wiki/wiki/Flakes#Enable_flakes)
+
+2. Set up [direnv](https://nix.dev/tutorials/declarative-and-reproducible-developer-environments#setting-up-direnv) - complete steps `1` and `2`
+
+3. Clone and step into this repo
+  ```sh
+  git clone https://github.com/cachix/cachix
+  cd cachix
+  ```
+
+4. Try using cache
+  ```sh
+  cachix use cachix-shell
```

Comment:
This should be `cachix use cachix` if anything

Reply:
I wonder how I missed this